src/custom-types.json

```diff
@@ -803,6 +803,10 @@
   "text/x-handlebars-template": {
     "extensions": ["hbs"]
   },
+  "text/x-java-properties": {
```

Comment:
Please add a sources key and include a primary source for the type data (https://github.com/jshttp/mime-db#contributing).

Reply:
Done